src/mono/browser/runtime/pthreads/browser/index.ts

```diff
@@ -217,3 +206,16 @@ export function dumpThreads(): void {
         console.info(`${idx} | ${info.threadPrefix}: isRunning:${isRunning} isAttached:${isAttached} isEventLoop:${isEventLoop} reuseCount:${reuseCount} - ${info.threadName}`);
     });
 }
+
+export function init_finalizer_thread() {
+    // we don't need it immediately, so we can wait a bit, to keep CPU working on normal startup
+    setTimeout(() => {
+        try {
+            cwraps.mono_wasm_init_finalizer_thread();
+        }
+        catch (err) {
+            mono_log_error("init_finalizer_thread() failed", err);
+            loaderHelpers.mono_exit(1, err);
+        }
+    }, 200);
```

Comment:
Can we do better that fixed number? I guess mobile phone VS our dev machines make difference

Reply:
`finalizerThreadStartDelayMs`